examples/02_meta-analyses/plot_cbma_combine_estimators_and_kernels.py

```diff
@@ -0,0 +1,172 @@
+# MKDA density analysis
+# --------------------------------------------------
+for kt_name, kt in kernel_transformers.items():
+    try:
+        mkda = nimare.meta.mkda.MKDADensity(kernel_transformer=kt)
+        mkda.fit(dset)
+        corr = nimare.correct.FWECorrector(
+            method="montecarlo", n_iters=10, n_cores=1
+        )
+        cres = corr.transform(mkda.results)
+        plot_stat_map(
+            cres.get_map("logp_level-voxel_corr-FWE_method-montecarlo"),
+            cut_coords=[0, 0, -8],
+            draw_cross=False,
+            cmap="RdBu_r",
+            title="MKDA estimator with %s" % kt_name,
+        )
+
+    except IndexError:
+        print(
+            "\nError: the %s does not currently work with the MKDA meta-analysis method\n"
+            % kt_name
+        )
+
+###############################################################################
+# MKDA Chi2
+# --------------------------------------------------
+for kt_name, kt in kernel_transformers.items():
+    try:
+        mkda = nimare.meta.mkda.MKDAChi2(kernel_transformer=kt)
+        dset1 = dset.slice(dset.ids)
+        dset2 = dset.slice(dset.ids)
+        mkda.fit(dset1, dset2)
+        corr = nimare.correct.FWECorrector(
+            method="montecarlo", n_iters=10, n_cores=1
+        )
+        cres = corr.transform(mkda.results)
+        plot_stat_map(
+            cres.get_map(
+                "z_desc-consistency_level-voxel_corr-FWE_method-montecarlo"
+            ),
+            threshold=1.65,
+            cut_coords=[0, 0, -8],
+            draw_cross=False,
+            cmap="RdBu_r",
+            title="MKDA Chi2 estimator with %s" % kt_name,
+        )
+
+    except IndexError:
+        print(
+            "\nError: the %s does not currently work with the MKDA Chi2 meta-analysis method\n"
+            % kt_name
+        )
+
+###############################################################################
+# KDA
+# --------------------------------------------------
+for kt_name, kt in kernel_transformers.items():
+    try:
+        kda = nimare.meta.mkda.KDA(kernel_transformer=kt)
+        kda.fit(dset)
+        corr = nimare.correct.FWECorrector(
+            method="montecarlo", n_iters=10, n_cores=1
+        )
+        cres = corr.transform(kda.results)
+        plot_stat_map(
+            cres.get_map("logp_level-voxel_corr-FWE_method-montecarlo"),
+            cut_coords=[0, 0, -8],
+            draw_cross=False,
+            cmap="RdBu_r",
+            title="KDA estimator with %s" % kt_name,
+        )
+
+    except IndexError:
+        print(
+            "\nError: the %s does not currently work with the KDA meta-analysis method\n"
+            % kt_name
+        )
+
+###############################################################################
+# ALE
+# --------------------------------------------------
+
+
+for kt_name, kt in kernel_transformers.items():
+<<<<<<< HEAD
```

Comment:
It looks like there's a merge conflict that got committed without being resolved.

Reply:
You're right, sorry I didn't catch that! I think I fixed it now.